lib/openfoodfacts.dart

```diff
@@ -855,4 +856,33 @@ class OpenFoodAPIClient {
       return KnowledgePanels.empty();
     }
   }
+
+  /// Returns the nutrient hierarchy specific to a country, localized.
+  ///
+  /// [cc] is the country code, as ISO 3166-1 alpha-2
+  static Future<OrderedNutrients?> getOrderedNutrients({
+    required final String cc,
+    required final OpenFoodFactsLanguage language,
+    final QueryType? queryType,
+  }) async {
+    final Uri uri = UriHelper.getUri(
+      path: 'cgi/nutrients.pl',
+      queryType: queryType,
+      queryParameters: <String, String>{'cc': cc, 'lc': language.code},
+    );
+
+    try {
+      final Response response = await HttpHelper().doGetRequest(
+        uri,
+        userAgent: OpenFoodAPIConfiguration.userAgent,
+      );
+      if (response.statusCode != 200) {
+        return null;
+      }
+      final json = jsonDecode(response.body);
+      return OrderedNutrients.fromJson(json);
+    } catch (exception) {
+      return null;
```

Comment:
We probably don't need to give detailed error messages since the package user doesn't provide any custom info themselves but at least when a error occurs we should throw the exception, shouldn't we?

Reply:
Looks good, feel free to merge